### PR TITLE
README beautification

### PR DESCRIPTION
**Pull request by mserranom**
- Added header with images
- Moved service list to a table

Rendered result: https://github.com/overleaf/overleaf/blob/msm-readme-restyle/README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,96 +1,68 @@
-Overleaf
-========
+<h1 align="center">
+  <br>
+  <a href="https://www.overleaf.com"><img src="https://images.ctfassets.net/nrgyaltdicpt/6gsvc5Ogjmu04I4Miu0uGg/cb1d4391717d2ab8d5e42ede6fb0eef1/overleaf_wide_colour_light_bg.png" alt="Overleaf" width="300"></a>
```
Comment (40thieves) [inline README.md]: Would https://www.overleaf.com/img/ol-brand/logo-horizontal.png work? I see that this is the contentful CDN, not sure we should be hotlinking to external assets.

File: README.md [hunk 2]
```diff
@@ -1,96 +1,68 @@
...
+</h1>
+
+<h4 align="center">An open-source online real-time collaborative LaTeX editor.</h4>
+
+<p align="center">
+  <a href="#key-features">Key Features</a> •
+  <a href="https://github.com/overleaf/overleaf/wiki">Wiki</a> •
+  <a href="https://www.sharelatex.com/i/university/onsite">Server Pro</a> •
```
Comment (40thieves) [inline README.md]: This seems to redirect to https://www.overleaf.com/for/enterprises? Are we planning on having a specific page for this?

File: README.md [hunk 3]
```diff
@@ -1,96 +1,68 @@
...
+  <a href="#contributing">Contributing</a> •
+  <a href="https://mailchi.mp/overleaf.com/community-edition-and-server-pro">Mailing List</a> •
+  <a href="#authors">Authors</a> •
+  <a href="#license">License</a>
+</p>
+
+<a href="https://www.overleaf.com"><img src="https://images.ctfassets.net/nrgyaltdicpt/b8LAh62mK5CAba7hrUFkK/b479c936bb524135a150703105d0930d/Overleaf-journal-template-source-trackchanges-example-hires.png" alt="Overleaf" ></a>
```
Comment (40thieves) [inline README.md]: Similarly how about https://www.overleaf.com/img/homepage@2x.png?

File: README.md (hunk 1 above)
Comment (mserranom) [reply README.md]: my fault, looking at `ctfassets` I took for granted it was a specific DNS service. I'll upload the images to Github, it's the best option.

File: README.md (hunk 3 above)
Comment (mserranom) [reply README.md]: Better and much lighter, thanks!

File: README.md (hunk 2 above)
Comment (mserranom) [reply README.md]: I copied over from other wiki links, didn't realise it was redirecting. Will replace with https://www.overleaf.com/for/enterprises.

File: README.md (hunk 1 above)
Comment (jdleesmiller) [reply README.md]: I have also been wondering about this... At the moment on the https://www.overleaf.com/for/partners/logos page we're telling people to link to assets on the CDN, so we're kind of committed to keeping them stable now. So... not sure.

File: README.md (hunk 2 above)
Comment (jdleesmiller) [reply README.md]: I think that is the best page to link to at the moment.

File: README.md (hunk 1 above)
Comment (jdleesmiller) [reply README.md]: We've been talking about retiring CDN77 and move to cloudflare, and I'm not sure that cdn.overleaf.com would actually be needed if we did that. Anyway... we control this, so it's not a big deal where it links to. As Miguel says, put the files on GitHub seems like a good solution (keeps it all in one place).

Comment (40thieves) [reply README.md]: I was more thinking of our guidelines on `/for/partners/logos`

> we control this, so it's not a big deal where it links to

Right, but not necessarily if contentful changes their implementation (I think we get these urls from the contentful api, so it in theory could change.

> the files on GitHub seems like a good solution

Agreed

Comment (pdrlps) [reply README.md]: It makes sense to unify the CDN solutions, specially if we're considering moving fonts out of Google Fonts and delivering FontAwesome ourselves.!

File: README.md (hunk 3 above)
Comment (mserranom) [reply README.md]: I'm actually keeping the same image, is the recommended in https://www.overleaf.com/for/press/media-resources (although will re-encode, it's 790KB)

**Pull request merged**